checklist-pl.md

```diff
@@ -0,0 +1,27 @@
+## 1. Dozwolona składnia markdown na Devchecklists:
+  * [ ] Github usuwa białe znaki podczas kompilacji plików markdown. Aby utworzyć poprawny plik skorzystaj z [nieprzetworzonej wersji](https://raw.githubusercontent.com/vintasoftware/devchecklists-template/master/checklist-pl.md) tego pliku.
+  * [ ] `##` dla tytułów.
+  * [ ] `* [ ]` składnia do stworzenia węzła głównego listy kontrolnej.
+  * [ ] `    * [ ]` składnia do stworzenia węzła drugiego poziomu listy kontrolnej.
+  * [ ] `    * np.` Składnia do stworzenia przykładu drugiego poziomu.
+  * [ ] `      * np.` Składnia do stworzenia przykładu trzeciego poziomu.
+  * [ ] Składnia dla prostego fragmentu kodu `Tutaj zamieść kod`.
+  * [ ] Składnia dla bloków kodu:
+    ```
+    # Blok kodu:
+	To jest
+	kawałek kodu
+	w bloku
+    ```
+
+## 2. Składnie, których nie należy używać:
+  * [ ] `      * [ ]` lista kontrolna trzeciego poziomu.
```

Comment:
Don't know if "lista kontrolna" would be suitable translation, for me it sounds weird